index.html

```diff
@@ -61,10 +61,8 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
           <div>
             <label><input type="radio" id="check3" class="checkbox" name="attribute" value="Not sure">Not sure</label>
           </div>

```

Comment:
Please use semantic HTML tags, this document would help you: [https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md](url)

Reply:
It's good feedback, Is true that is important to bring some options to understand and fix the problem
Thank you